### PR TITLE
Dropzones searches and stuff

### PR DESCRIPTION
**Pull request by tchorzewski1991**

### CODE REVIEW

File: db/views/simple_searches_v01.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  'Artist' AS searchable_type
+FROM artists
+
+UNION
+SELECT labels.id AS searchable_id, labels.name AS term,
+  'Label' AS searchable_type
+FROM labels
+
+UNION
+SELECT record_formats.id AS searchable_id, record_formats.name AS term,
```
Comment (holden) [inline db/views/simple_searches_v01.sql]: Search record formats doesn't seem useful.

Comment (holden) [inline db/views/simple_searches_v01.sql]: At least not in the aggregate

Comment (tchorzewski1991) [inline db/views/simple_searches_v01.sql]: ok, I will remove this

File: db/views/simple_searches_v01.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  'RecordFormat' AS searchable_type
+FROM record_formats
+
+UNION
+SELECT genres.id AS searchable_id, genres.name AS term,
+  'Genre' AS searchable_type
+FROM genres
+
+UNION
+SELECT records.id AS searchable_id, records.comment AS term,
```
Comment (holden) [inline db/views/simple_searches_v01.sql]: This should likely be an aggregated field, perhaps joining with prices.detail + footnote?

Outer join, as not all records have a price.

Comment (holden) [inline db/views/simple_searches_v01.sql]: There's a bit of this one festiment or question one's search (aggregating terms with spaces)

File: db/views/simple_searches_v01.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
+SELECT artists.id AS searchable_id, artists.name AS term,
+  'Artist' AS searchable_type
```
Comment (holden) [inline db/views/simple_searches_v01.sql]: we might also wanna join all of these with record to remove blanks.  Not sure how useful non existent results are.

**Pull request merged**